azure-pipelines.yml

```diff
@@ -95,24 +102,56 @@ stages:
           secureFile: SonarSourceSecret.snk
 
       - task: DownloadSecureFile@1
-        displayName: 'Download pfx file'
-        name: pfx
+        displayName: 'Download p12 file'
+        name: SM_CLIENT_CERT
         inputs:
-          secureFile: 'SonarSource-2021-2023.pfx'
+          secureFile: digicert_authentication_certificate.p12
+ 
+      - task: DownloadSecureFile@1
+        displayName: 'Download crt file'
+        name: SM_CLIENT_CRT
+        inputs:
+          secureFile: cert_525594307.crt
+
+      - task: PowerShell@2
+        displayName: "Signing certificate setup"
+        # ToDo: add back the condition
+        # condition: eq(variables.isReleaseBranch, 'True')
```

Comment:
I will remove the ToDos once the PR is "ready" to be merged so we can validate the signing until then